src/RefactoringGuru/Flyweight/RealWorld/FlyweightRealWorld.php

```diff
@@ -131,12 +203,21 @@ public function __construct($name, $age, $owner, CatVariation $variation)
     }
 
     /**
+     * EN:
      * Since the Context objects don't own all of their state, sometimes you'll
      * need to provide some convenient helper method (for comparing their data
      * with other objects, for example).
      *
      * @param $query
      * @return bool
+     *
+     * RU:
+     * Поскольку объекты Контекста не владеют всем своим состоянием, иногда вам 
```

Comment:
Тут в оригинале криво написано, думаю, стоит перефразировать так:

Since the Context objects don't own all of their state, sometimes, for the sake of convenience, you may need to implement some helper methods (for example, for comparing several Context objects.)

Поскольку объекты Контекста не владеют всем своим состоянием, иногда для удобства вы можете реализовать несколько вспомогательных методов (например, для сравнения нескольких объектов Контекста между собой).